### PR TITLE
Mar 694 - Public barrier list API enhancements

### PR DESCRIPTION
**Pull request by jamespacileo**

### CODE REVIEW

File: api/barriers/serializers/public_barriers.py [hunk 1]
```diff
@@ -183,20 +188,30 @@ def get_internal_code(self, obj):
     def get_internal_id(self, obj):
         return obj.barrier_id
 
+    def get_internal_government_organisations(self, obj):
+        return OrganisationSerializer(obj.barrier.organisations, many=True).data
+
     def get_latest_note(self, obj):
         try:
-            note = obj.notes.latest("created_on")
+            # We need to perform Python sorting instead of SQL
```
Comment (rossmiller) [inline api/barriers/serializers/public_barriers.py:196]: Could you use .first() here?

**Review by rossmiller — COMMENTED** [review]
One comment to address.

**Review by rossmiller — APPROVED** [review]
LGTM!

File: api/barriers/serializers/public_barriers.py (hunk 1 above)
Comment (jamespacileo) [reply api/barriers/serializers/public_barriers.py:196]: Had to avoid `first()` unfortunately as it overrides prefetct_related and triggers a new query

**Pull request merged**